ui/app/src/components/workspaces/WorkspaceUsers.tsx

```diff
@@ -1,24 +1,38 @@
-import * as React from "react";
-import { useState, useCallback, useEffect, useMemo, useContext } from "react";
-import { GroupedList, IGroup } from "@fluentui/react/lib/GroupedList";
-import { IColumn, DetailsRow } from "@fluentui/react/lib/DetailsList";
-import { SelectionMode } from "@fluentui/react/lib/Selection";
-import { Persona, PersonaSize } from "@fluentui/react/lib/Persona";
-import { HttpMethod, useAuthApiCall } from "../../hooks/useAuthApiCall";
-import { APIError } from "../../models/exceptions";
-import { WorkspaceContext } from "../../contexts/WorkspaceContext";
-import { ApiEndpoint } from "../../models/apiEndpoints";
-import { LoadingState } from "../../models/loadingState";
-import { ExceptionLayout } from "../shared/ExceptionLayout";
-import { User } from "../../models/user";
-import { Stack } from "@fluentui/react";
+import * as React from 'react';
+import { useState, useCallback, useEffect, useMemo, useContext } from 'react';
+import { GroupedList, IGroup } from '@fluentui/react/lib/GroupedList';
+import { IColumn, DetailsRow } from '@fluentui/react/lib/DetailsList';
+import { SelectionMode, Selection, SelectionZone } from '@fluentui/react/lib/Selection';
+import { Persona, PersonaSize } from '@fluentui/react/lib/Persona';
+import { HttpMethod, useAuthApiCall } from '../../hooks/useAuthApiCall';
+import { APIError } from '../../models/exceptions';
+import { WorkspaceContext } from '../../contexts/WorkspaceContext';
+import { ApiEndpoint } from '../../models/apiEndpoints';
+import { LoadingState } from '../../models/loadingState';
+import { ExceptionLayout } from '../shared/ExceptionLayout';
+import { User } from '../../models/user';
+import { AppRolesContext } from '../../contexts/AppRolesContext';
+
+import { CommandBarButton, DefaultButton, Dialog, DialogFooter, getTheme, Spinner, SpinnerSize, Stack } from '@fluentui/react';
+import { useNavigate, Route, Routes } from 'react-router-dom';
+import { destructiveButtonStyles } from '../../styles';
+import { WorkSpaceUsersAssignNew } from './WorkspaceUsersAssignNew';
+import config from "../../config.json"
```

Comment:
This file churn is converting double-quotes to single-quotes.
Did something change in the way we lint and format these files, or is this a personal choice/local editor preference?

I'm not a typescript person, but I thought double quotes were preferred. Either way we don't want this to flip-flop for each developer.

Can we make sure that the tooling in the devcontainer enforces a single approach, and double check if this is that approach?

Reply:
To be clear, I'd rather see less churn, not more, in this PR.

i.e. it would be better to see this PR focus on adding the functionality (i.e. fewer changes to things like quotes) rather than to expand to include additional format changes.

Let's move the question of "what standard should we be using for typescript" into another PR.